specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -492,6 +635,203 @@
           }
         }
       }
+    },
+    "/weather/tropical/storms/active/{format}": {
+      "get": {
+        "description": "**Get Active Storms**\n\n**Applies to**: S0 and S1 pricing tiers.\n\nGet all government-issued active storms.",
```

Comment:
To be consistent with how all the Azure Maps Docs pages are going to be updated soon, please change to:

Gen 1 (S0 and S1) and Gen 2 pricing tiers.

Reply:
Updated